segmentation/utils/spatial_analysis_utils.py

```diff
@@ -110,56 +114,65 @@ def compute_close_cell_num(dist_mat, dist_lim, num, analysis_type,
     cell_label_col = "cellLabelInImage"
 
     # Subset data based on analysis type
-    if analysis_type == "Channel":
+    if analysis_type == "channel":
         # Subsetting the column with the cell labels
-        cell_labels = fov_data[cell_label_col]
+        cell_labels = current_fov_data[cell_label_col]
 
     # Create close_num, marker1_num, and marker2_num
     close_num = np.zeros((num, num), dtype='int')
-    mark1_num = []
-    mark1poslabels = []
-
-    dist_mat_bin = np.zeros(dist_mat.shape, dtype='int')
-    dist_mat_bin[dist_mat < dist_lim] = 1
+    marker1_poslabels_num = []
+    marker2_poslabels_num = []
```

Comment:
revert to removed lines 119-123 but change `mark1poslabels` to `marker1_poslabels_num`